From 043b982c4f8a87106c4ad42eea2fd0072b76ac1a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 16:38:09 +0300
Subject: [PATCH] Makefile: start the release jenkins job

Finally, after pushing the release tags, start the jenkins release build.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index ec2e079..f175390 100644
--- a/Makefile
+++ b/Makefile
@@ -75,6 +75,8 @@
 	@repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
+	@wget http://192.168.0.4:8080/job/goya%20release%20test/build?token=goya-release
+	@echo "Started the \"goya release test\" jenkins job... check your email for results."
 
 .PHONY: hl-dali hl-gaudi
 hl-dali hl-gaudi: $(OUTPUT_DIR)
